MantidQt/CustomInterfaces/src/Muon/ALCHelper.cpp

```diff
@@ -28,6 +28,25 @@ namespace ALCHelper
     return boost::make_shared<QwtArrayData>(x,y,size);
   }
 
+  std::vector<boost::shared_ptr<QwtData>>
+  curveDataFromWs(MatrixWorkspace_const_sptr ws) {
+
+    std::vector<boost::shared_ptr<QwtData>> dataVector;
+    auto histograms = ws->getNumberHistograms();
+
+    for (size_t wsIndex = 0; wsIndex < histograms; wsIndex++) {
+
+      const double *x = &ws->readX(wsIndex)[0];
+      const double *y = &ws->readY(wsIndex)[0];
+      size_t size = ws->blocksize();
+
+      auto wsIdxData = boost::make_shared<QwtArrayData>(x, y, size);
+
+      dataVector.push_back(wsIdxData);
+    }
+    return dataVector;
+  }
+
```

Comment:
This modifies a helper that belongs to a different interface. I'd put a comment saying that this is here for the Engg interface at least. Ideally this should go to a different place.